static/style.css

```diff
@@ -103,11 +103,14 @@ header h2 {
 }
 /* --------------- Nav --------------- */
 
-.navbar {
+.navbar, .navbar-style {

```

Comment:
`.navbar` and `.navbar-style` feel like they're the same thing. I'd change the names here to be more descriptive or consolidate them. Do you need to separate them?

Reply:
kinda, one is the actual navbar and the other is just the navbar styling. I want some elements to look like the navbar but not behave like it. Not the best UI design at the moment. Will keep that in mind for the next overhaul.